packages/paste-core/components/typography/src/Heading.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import {Text} from '@twilio-paste/text';
+
+export type asTags = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'div' | 'label' | 'span';
+export type HeadingStyle =
+  | 'headingStyle10'
+  | 'headingStyle20'
+  | 'headingStyle30'
+  | 'headingStyle40'
+  | 'headingStyle50'
+  | 'headingStyle60';
+
+interface Heading {
+  as: asTags;
+  id?: never;
+  className?: never;
+  children: NonNullable<React.ReactNode>;
+  headingStyle?: HeadingStyle;
+}
+
+function getHeadingStyles(headingStyle?: HeadingStyle): {} {
+  switch (headingStyle) {
+    case 'headingStyle60':
+      return {
+        marginBottom: 'space70',
+        fontSize: 'fontSize90',
+        fontWeight: 'fontWeightSemibold',
+        lineHeight: 'lineHeight60',
+      };
+    case 'headingStyle50':
```

Comment:
I _think_ you can do 

```
case 'headingStyle50':
default:
```

and delete the default from the bottom. This way we always know a default is a defined heading style.